app/lib/actions/pulp/repository/create.rb

```diff
@@ -31,6 +31,9 @@ def run
                                                     importer,
                                                     distributors,
                                                     display_name: input[:name])
+        rescue RestClient::Conflict
+          Rails.logger.warn("Tried to add repository #{input[:pulp_id]} that already exists.")
+          []
```

Comment:
I'm sorry to be a bit late for the review party here, but keeping some flag inside `output` could be helpful for investigating what actually happened inside the task later, without need for cross-checking with production.log